Aqovia.PactProducerVerifier/ProducerVerifierConfiguration.cs

```diff
@@ -8,6 +9,7 @@ public class ProducerVerifierConfiguration
         public string PactBrokerUsername { get; set; }
         public string PactBrokerPassword { get; set; }
         public string PactBrokerUri { get; set; }
+        public Func<IWebHostBuilder> GetBaseWebHostBuilder { get; set; } = Microsoft.AspNetCore.WebHost.CreateDefaultBuilder;
```

Comment:
Does this mean that when a consumer sets GetBaseWebHostBuilder that a (wasted) default initiliasation of a WebHost still happens anyway?
Or if we instantiate the class by something like `var config = new ProducerVerifierConfiguration (GetBaseWebHostBuilder = foo); does that mean that default never gets initialised? (Gap in my knowledge).

Reply:
@ma499 as i understand its the default if its not set

Reply:
@ma499 In regards to keeping it in sync, IWebHostBuilder is only relevant for aspnet core so wouldn't apply to the main branch, so please ignore my comment for this specific pull request. I'm approving the pull request

Reply:
> Does this mean that when a consumer sets GetBaseWebHostBuilder that a (wasted) default initiliasation of a WebHost still happens anyway

Nope, thanks to the property being just a function, which allows us to just pass the `CreateDefaultBuilder` method group.